src/ThreeJsPanel.ts

```diff
@@ -184,6 +184,8 @@ export class ThreeJsPanel {
     if (parentElement) {
       this.renderer.setSize(parentElement.offsetWidth, parentElement.offsetHeight);
       this.meshRenderTarget.setSize(parentElement.offsetWidth, parentElement.offsetHeight);
+      this.meshRenderTarget.depthTexture.dispose();
+      this.meshRenderTarget.depthTexture = new DepthTexture(parentElement.offsetWidth, parentElement.offsetHeight);
     }
```

Comment:
It looks like this whole `if` statement might be redundant? The depth texture is initially created with dimensions from `this.canvas.(width/height)`, and if `parentElement` exists, those are set to `parentElement.offset(Width/Height)`.